Documentation/Books/Manual/Administration/Arangoimport.md

```diff
@@ -450,3 +450,43 @@ Importing the following document will then create an edge between *users/1234* a
 ```js
 { "_from" : "1234", "_to" : "4321", "desc" : "users/1234 is connected to products/4321" }
 ```
+
+
+### Arangoimport with busy or low throughput disk subsystems
+
+Arangoimport has an automatic pacing algorithm that limits how fast
+data is sent to the ArangoDB servers.  This pacing algorithm exists to
+prevent the import operation from failing due to slow responses.
+
+Google Compute and other VM providers limit the throughput of disk
+devices. Google's limit is more strict for smaller disk rentals, than
+for larger. Specifically, a user could choose the smallest disk space
+and be limited to 3 Mbytes per second.
+
+The pacing algorithm adjusts the transmit block size dynamically based
+upon the actual throughput of the server over the last 20
+seconds. Further, each thread delivers its portion of the data in
+mostly non-overlapping chunks. The thread timing creates intentional
+windows of non-import activity to allow the server extra time for meta
+operations.
+
+The pacing algorithm works successfully with mmfiles with disks
+limited to read and write throughput as small as 1 Mbyte per
+second. The algorithm works successfully with rocksdb with disks
+limited to read and write throughput as small as 3 Mbyte per second.
+
+This algorithm will slow import of an unlimited (really fast) disk
```

Comment:
slow down the import - but lets rather talk about 'not utilizing all available bandwith' here. we should not use 'slow' in conjunction with arangodb.